core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/htmlviz/HtmlVizOpExecConfig.scala

```diff
@@ -1,39 +0,0 @@
-package edu.uci.ics.texera.workflow.operators.visualization.htmlviz
```

Comment:
any reason why this is removed?

Reply:
we are unifying config files. it is not necessary to create multiple config files.